eclair-core/src/main/scala/fr/acinq/eclair/db/PaymentsDb.scala

```diff
@@ -16,40 +16,70 @@
 
 package fr.acinq.eclair.db
 
+import java.util.UUID
+
 import fr.acinq.bitcoin.ByteVector32
 
 /**
-  * Store the Lightning payments received by the node. Sent and relayed payments are not persisted.
+  * Store the Lightning payments received and sent by the node. Relayed payments are not persisted.
   * <p>
-  * A payment is a [[Payment]] object. In the local context of a LN node, it is safe to consider that
+  * A received payment is a [[ReceivedPayment]] object. In the local context of a LN node, it is safe to consider that
   * a payment is uniquely identified by its payment hash. As such, implementations of this database can use the payment
   * hash as a unique key and index.
   * <p>
+  *
+  * <p>
+  * A sent payment is a [[OutgoingPayment]] object.
+  * <p>
   * Basic operations on this DB are:
   * <ul>
   * <li>insertion
   * <li>find by payment hash
   * <li>list all
   * </ul>
-  * Payments should not be updated nor deleted.
   */
 trait PaymentsDb {
 
-  def addPayment(payment: Payment)
+  def addReceivedPayment(payment: ReceivedPayment)
+
+  def addSentPayments(sent: OutgoingPayment)
+
+  def updateOutgoingStatus(id: UUID, newStatus: OutgoingPaymentStatus.Value)
+
+  def receivedByPaymentHash(paymentHash: ByteVector32): Option[ReceivedPayment]
 
-  def findByPaymentHash(paymentHash: ByteVector32): Option[Payment]
+  def sentPaymentById(id: UUID): Option[OutgoingPayment]
 
-  def listPayments(): Seq[Payment]
+  def sentPaymentByHash(paymentHash: ByteVector32): Option[OutgoingPayment]
 
-  def close: Unit
+  def listReceived(): Seq[ReceivedPayment]
+
+  def listSent(): Seq[OutgoingPayment]
 
 }
 
 /**
-  * Payment object stored in DB.
+  * Received payment object stored in DB.
   *
-  * @param payment_hash identifier of the payment
+  * @param paymentHash identifier of the payment
+  * @param amount_msat amount of the payment, in milli-satoshis
+  * @param timestamp   absolute time in seconds since UNIX epoch when the payment was created.
+  */
+case class ReceivedPayment(paymentHash: ByteVector32, amountMsat: Long, timestamp: Long)
+
+/**
+  * Outgoing payment is every payment that is sent by this node, they may not be finalized and
+  * when is final it can be failed or successful.
+  *
+  * @param id           internal payment identifier
+  * @param payment_hash payment_hash
   * @param amount_msat  amount of the payment, in milli-satoshis
-  * @param timestamp    absolute time in seconds since UNIX epoch when the payment was created.
+  * @param lastUpdate   absolute time in seconds since UNIX epoch when the payment was last updated.
   */
-case class Payment(payment_hash: ByteVector32, amount_msat: Long, timestamp: Long)
+case class OutgoingPayment(id: UUID, paymentHash: ByteVector32, amountMsat: Long, lastUpdate: Long, status: OutgoingPaymentStatus.Value)
```

Comment:
```suggestion
case class SentPayment(id: UUID, paymentHash: ByteVector32, amountMsat: Long, lastUpdate: Long, status: SentPaymentStatus.Value)
```

For consistency with `ReceivedPayment`? I get that the payment can fail, but it is still sent and the resolution doesn't depend on the sender.

And also this is not consistent with `listSent`